pkgs/tools/misc/pandoc-drawio-filter/default.nix

```
@@ -0,0 +1,36 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, drawio
+, pandocfilters
+, xvfb-run
+, lib
+}:
+
+let
+  tag = "1.0";
+in
+
+buildPythonApplication {
+  name = "pandoc-drawio-filter";
+  version = tag;
+
+  src = fetchFromGitHub {
+    owner = "tfc";
+    repo = "pandoc-drawio-filter";
+    rev = tag;
```

```suggestion
buildPythonApplication rec {
  name = "pandoc-drawio-filter";
  version = "1.0";

  src = fetchFromGitHub {
    owner = "tfc";
    repo = "pandoc-drawio-filter";
    rev = version;
```

Thanks for adding a tag, I was going to say that it's preferred, yes.

---

Why would you bring `name` and `src` into scope?

---

What do you mean?

---

i prefer to avoid `rec` attrsets.

---

> What do you mean?

That's the effect of your change. You've added a keyword that has the effect of bringing those identifiers into scope for subexpressions that don't need it.

I guess it "works", but it doesn't convey intent the way a `let` binding does.